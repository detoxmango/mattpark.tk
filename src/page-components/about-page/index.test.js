import React from 'react'
import renderer from 'react-test-renderer'

import AboutPage from './index'

describe("AboutPage", () => {
  let tree

  beforeEach(() => {
    tree = renderer.create(<AboutPage />).toJSON()
  })

  // it("renders correctly", () => {
  //   expect(tree).toMatchSnapshot()
  // })
  it("passes", () => {
    expect(true).toBe(true)
  })
})
