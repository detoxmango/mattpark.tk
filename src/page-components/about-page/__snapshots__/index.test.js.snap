// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage renders correctly 1`] = `
Array [
  <h1
    className="MuiTypography-root MuiTypography-h1"
  >
    About Me
  </h1>,
  <h4
    className="MuiTypography-root MuiTypography-h4"
  >
    Under construction..
  </h4>,
]
`;
